packages/schema/src/model.ts

```diff
@@ -458,6 +496,29 @@ export class JupyterGISModel implements IJupyterGISModel {
     this.sharedModel.layerTree = layerTree;
   }
 
+  private _removeLayerTreeGroup(
+    layerTree: IJGISLayerItem[],
+    groupName: string
+  ) {
+    // Iterate over each item in the layerTree
+    for (let i = 0; i < layerTree.length; i++) {
+      const currentItem = layerTree[i];
+
+      // Check if the current item is a string and matches the target
+      if (typeof currentItem !== 'string' && currentItem.name === groupName) {
+        // Remove the item from the array
+        layerTree.splice(i, 1);
+        // Decrement i to ensure the next iteration processes the remaining items correctly
+        i--;
+      } else if (typeof currentItem !== 'string' && 'layers' in currentItem) {
+        // If the current item is a group, recursively call the function on its layers
+        this._removeLayerTreeGroup(currentItem.layers, groupName);
+      }
+    }
+
+    this.sharedModel.layerTree = layerTree;
+  }
+
```

Comment:
If `_removeLayerTreeLayer` and `_removeLayerTreeGroup` are the same we could probably just have one `_removeLayerTreeItem` function.